x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/searchablesnapshots/MountSearchableSnapshotRequest.java

```diff
@@ -109,6 +110,22 @@ public MountSearchableSnapshotRequest(StreamInput in) throws IOException {
         }
     }
 
+    /**
+     * Returns the preference for new searchable snapshot indices. When
+     * performing a full mount the preference is cold - warm - hot. When
+     * performing a partial mount the preference is only frozen
+     */
+    public static String getDataTiersPreference(Storage type) {
```

Comment:
nit/qq: why not an instance method on `Storage`?

Reply:
Doh, thanks, fixed in 32371ac45b6220d63bcaafdedc9b38f6ca4d5440